### PR TITLE
Update the public options of customform

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
The requirements changed and users can publish a form via URL but they don't have to publish it for the whole organization.

## Approach
- I added a JSON field to provide a way to persist the public options in a combined way

### Furter Info
Related ticket: https://github.com/Humanitec/ActivityAPI/issues/441

### CODE REVIEW

**Comment by Menda** [comment]
`I added a HStore field to provide a way to persist the public options in a combined way`

I'd say it's a JSON

File: formlibrary/signals.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.dispatch import receiver
+from django.db.models.signals import pre_save
+from .models import CustomForm
+
+
+@receiver(pre_save, sender=CustomForm)
+def pre_save_handler(sender, instance, *args, **kwargs):
+    instance.full_clean()
```
Comment (Menda) [inline formlibrary/signals.py]: I guess it's a thing about preference, but to follow better the flow, I would have the `full_clean()` call inside the `save()`. That's how we are doing it in our microservices and the recommended way in the Django documentation.

Comment (jefmoura) [reply formlibrary/signals.py]: It's not like that in the `contacts`
https://github.com/Humanitec/ActivityAPI/blob/master/contact/signals.py#L8

Comment (jefmoura) [reply formlibrary/signals.py]: Neither `appointments`
https://github.com/Humanitec/ActivityAPI/blob/master/appointment/signals.py#L8

Comment (jefmoura) [reply formlibrary/signals.py]: Neither in `documents-service`
https://github.com/Humanitec/documents-service/blob/master/documents/signals.py#L9

**Comment by jefmoura** [comment]
At the beginning, I added HStore but it wasn't working.. that's why :sweat_smile:

File: formlibrary/signals.py (hunk 1 above)
Comment (Menda) [reply formlibrary/signals.py]: Yeah.. just this guy: https://github.com/Humanitec/time_tracking_service/blob/master/timetracking/models.py#L43

**Pull request merged**